mentor_classifier/tests/helpers.py

```diff
@@ -5,6 +5,110 @@
 # The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
 #
 from os import path
+from dataclasses import dataclass, asdict, field
+from typing import List, Optional
+
+
+@dataclass
+class MentorQuestion:
+    question_id: str
+    question: str
+    paraphrases: List[str]
+    answer: str
+    topic: str
+
+
+@dataclass
+class Subject:
+    name: str
+
+
+@dataclass
+class Topic:
+    name: str
+
+
+@dataclass
+class Question:
+    _id: str
+    question: Optional[str] = None
+    type: Optional[str] = None
+    name: Optional[str] = None
+    paraphrases: List[str] = field(default_factory=list)
+
+
+@dataclass
+class Answer:
+    _id: str
+    status: str
+    transcript: str
+    question: Question
+
+
+@dataclass
+class QuestionAndTopics:
+    question: Question
+    topics: List[Topic]
+
+
+@dataclass
+class Mentor:
+    subjects: List[Subject] = field(default_factory=list)
+    topics: List[Topic] = field(default_factory=list)
+    questions: List[QuestionAndTopics] = field(default_factory=list)
+    answers: List[Answer] = field(default_factory=list)
+
+    def to_dict(self):
+        return asdict(self)
+
+    def add_mentor_question(self, mentor_question: MentorQuestion):
+        question_ref = Question(_id=mentor_question.question_id)
+        topic = Topic(mentor_question.topic)
+        question_and_topics = QuestionAndTopics(question=question_ref, topics=[topic])
+        if topic not in self.topics:
+            self.topics.append(topic)
+        self.questions.append(question_and_topics)
+        question = Question(
+            _id=mentor_question.question_id,
+            question=mentor_question.question,
+            type=None,
+            name=None,
+            paraphrases=mentor_question.paraphrases,
+        )
+        answer = Answer(
+            _id="A" + str(len(self.answers) + 1),
+            status="COMPLETE",
+            transcript=mentor_question.answer,
+            question=question,
+        )
+        self.answers.append(answer)
+
+
+def load_mentor_csv(path: str) -> Mentor:
+    result = Mentor()
+    with open(path) as f:
+        lines = f.readlines()
+        lines.pop(0)
+
+        for line in lines:
+            mentor_question = parse_mentor_question(line)
+            result.add_mentor_question(mentor_question)
+
+    return result
+
+
+def parse_mentor_question(csv_line: str) -> MentorQuestion:
+    columns = csv_line.split(",")
+    paraphrases = []
+    if columns[2] != "":
```

Comment:
always prefer falsy check

```python
if not columns[2]:
```

...but here you should not use this `if` stmt at all, just do

```python
paraphrases = columns[2].split("|") if columns[2] else []
```